plugin/clang.vim

```diff
@@ -840,10 +840,40 @@ func! s:ClangCompleteInit(force)
 endf
 "}}}
 "{{{ ClangExecuteNeoJobHandler
-"handles event: exit
+"Handles events: stdout, stderr, exit
 func! ClangExecuteNeoJobHandler(job_id, data, event)
-  if a:event == 'exit'
-    call ClangExecuteDone(self.fstdout, self.fstderr)
```

Comment:
Very nice improvement, but still can be more.
You can see here the code for `stderr` and `stdout` are all same except for the string `'stdout'` and `'stderr'`. So I recommend to use `a:event` to replace the string and can reduce half of code.